### DISCUSSION TITLE
Duda variable color

### DISCUSSION THREAD

**Discussion by IgnacioHqz** [IIC2133-PUC/2024-2#10] [Tarea 0] [answered]
Hola! Tengo una duda con respecto al color de los puffles, los colores son char pero son de solo un carácter? no sé si lo estoy entendiendo bien para definirlo correctamente en la estructura.
Gracias!

**Comment by tomastrivino** [comment, marked as answer]
Hola,
como se observa en los distintos input, los colores que se entregan son de múltiples caracteres. Es decir, el color que se te entrega es algo de la forma `green` en vez de `g`, por ejemplo. En el código base se define con `char color`, pero recuerda que lo puedes modificar de la manera que estimes conveniente para realizar tu solución.

**Reply by tomastrivino** [reply]
Acá están las posibilidades de todos modos: 
<img width="795" alt="image" src="https://github.com/user-attachments/assets/0629288c-f225-4c58-ab8b-91de999e00e6">

**Reply by IgnacioHqz** [reply]
yaa perfecto, se me había olvidado que se había hecho esa aclaración de los 10 colores.
Muchas gracias por la aclaración!

**Reply by tomastrivino** [reply]
Bacán! cualquier cosa pregunta nomás
PD. Buena foto de perfil
